src/core/oauth2-authorize.js

```diff
@@ -37,7 +37,7 @@ export default function authorize ( { auth, authActions, errActions, configs, au
   }
 
   url += "&redirect_uri=" + encodeURIComponent(redirectUrl)
-      + "&realm=" + encodeURIComponent(authConfigs.realm);
+      + "&realm=" + encodeURIComponent(authConfigs.realm)
```

Comment:
this change fixes: https://github.com/swagger-api/swagger-ui/commit/8aebea34c4ee1f063a8a5c6800f0d8643751eff3